templates/emails/onboarding.day7.hbs

```diff
@@ -1,16 +1,21 @@
-Subject: Give visibility to your backers and sponsors
+Subject: Engage your community on Open Collective
 
 {{> header}}
 
 <h1>Hey {{collective.name}} 👋</h1>
 
-<p>Did you know that we offer widgets to automatically show your backers and sponsors on your website? It's a great way to thank them for their financial contribution. It will also incentivize others to contribute as well!</p>
+<p>Did you know we offer some neat features to better engage with and show gratitude to your community?</p>
 
-<p>Go to the settings of your collective in the "export" tab: <a href="https://opencollective.com/{{collective.slug}}/edit/export">https://opencollective.com/{{collective.slug}}/edit/export</a></p>
+<ul>
+  <li> <a href="https://docs.opencollective.com/help/collectives/communication">Send Updates</a>: Let your backers know what their support is enabling. Updates go out as an email and also automatically show on your Collective page.</li>
+  <li> <a href="https://docs.opencollective.com/help/collectives/conversations">Start a conversation</a>: They are public threads that everyone can read and participate to. Use them to get feedback from your community or to organize your actions!</li> 
+  <li> <a href="https://docs.opencollective.com/help/collectives/integrations#twitter-integration">Connect Twitter</a>: Automatically send thank you Tweets to new backers, and announce your Collective’s monthly stats.</li>
+  <li> <a href=:https://docs.opencollective.com/help/collectives/integrations#zapier-integration"> Zapier App</a>: The best choice whenever you need to integrate Open Collective with a third party service.</li>
+  <li> <a href="https://docs.opencollective.com/help/collectives/widgets">Website widgets</a>: Show gratitude to your sponsors by displaying them on your website. You can add a ‘donate’ button, too.</li>
+  <li> <a href="https://github.com/opencollective/opencollective-bot">Open Collective bot</a>: Automate interactions on Github linked back to your Collective, such as prioritizing issues raised by your backers and sponsors.</li>
```

Comment:
```suggestion
  <li> <a href="https://github.com/opencollective/opencollective-bot">Open Collective bot</a>: Automate interactions on Github repos linked back to your Collective, such as prioritizing issues raised by your backers and sponsors.</li>
```

Reply:
I personally try to use `repositories` instead of `repos`, find it less "jargon".